spring-boot-project/spring-boot-docs/src/docs/asciidoc/web/spring-security.adoc

```diff
@@ -223,7 +223,19 @@ Again, the same properties are applicable for both servlet and reactive applicat
 
 Alternatively, you can define your own `OpaqueTokenIntrospector` bean for servlet applications or a `ReactiveOpaqueTokenIntrospector` for reactive applications.
 
+The OAuth2 resource server now supports validating audience (aud) claims in JSON Web Tokens (JWTs) issued by an authorization server. 
+This feature is added to complement the existing validation of issuer (iss) claims.
 
+To enable audience validation, set the `spring.security.oauth2.resourceserver.jwt.audience` property in your Spring Boot application 
```

Comment:
This property name is incorrect. It should be `spring.security.oauth2.resourceserver.jwt.audiences`. As previously requested, please use the `configprop:spring.security.oauth2.resourceserver.jwt.audiences[]` syntax so that the name is checked as part of the build.